radar_parlamentar/plenaria/ordenacao.py

```diff
@@ -91,7 +91,7 @@ def ordenar_partidos(casa, periodo):
     analise_periodo = analisador.analisa()
     coords = analise_periodo.coordenadas_partidos
     coords_list = []
-    for k in coords.iteritems():
+    for k in list(coords.items()):
```

Comment:
Nós não precisamos do `list()` aqui.
```for k in coords.items():``` é a mesma coisa que ```for k in list(coords.items()):```

Mais do que isso, esse `for` é completamente desnecessário.
Podemos trocar das linhas 92 a 96 por:
```python
coords = analise_periodo.coordenadas_partidos
partidos = sorted([(x, y[0]) for x, y in coords_list.items()],
                  key=itemgetter(1,0)
```